### PR TITLE
update cams_glc_2017 yaml

### PR DESCRIPTION
**Pull request by chorng**

### CODE REVIEW

**Comment by chorng** [comment]
The old pull request had a conflict as new commit being merged. This data collection has been migrated to aws 3 months ago and the collection id showing on the website atm is outdated.

File: collections/cams_glc_2017.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 Name: CAMS_GLC_2017
-OpenEOPID: CAMS_GLC
```
Comment (dthiex) [inline collections/cams_glc_2017.yaml:2]: Why did you remove this? It's needed to generate the ID for https://collections.eurodatacube.com/stac/index.json. We should probably update the contributing.md file with that info.

File: collections/cams_glc_2017.yaml [hunk 2]
```diff
@@ -51,7 +50,7 @@ Extent:
     interval:
       -
         - '2017-01-01T00:00:00Z'
-        - '2017-01-01T00:00:01Z'
+        - '2017-12-31T00:00:00Z'
```
Comment (dthiex) [inline collections/cams_glc_2017.yaml:53]: I would strongly vote for leaving the old extend here. We updated this for all collections [here](https://github.com/eurodatacube/public-collections/pull/218) because it's quite confusing that we have the whole year as extend but then the collection is only available on the first of January. The underlying problem is a question between what data went in to create the collection/the collection is showing and for what dates does the collection return actually data. Something that we should also document better.

File: collections/cams_glc_2017.yaml [hunk 3]
```diff
@@ -69,12 +68,12 @@ CubeDimensions:
...
+      - '2017-12-31T00:00:00Z'
   bands:
     type: bands
     values:
       - RED
       - BLUE
       - GREEN
 RegistryEntryAdded: "2021-05-07"
-RegistryEntryLastModified: "2022-07-19"
+RegistryEntryLastModified: "2022-02-10"
```
Comment (dthiex) [inline collections/cams_glc_2017.yaml:79]: ?

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/cams_glc_2017.yaml (hunk 1 above)
Comment (chorng) [reply collections/cams_glc_2017.yaml]: Sorry, I copied the file I edited several months ago. I put it back.

File: collections/cams_glc_2017.yaml (hunk 2 above)
Comment (chorng) [reply collections/cams_glc_2017.yaml]: Sorry, this file was copied from the one I edited several months ago. I didn't know the changes. Fix to fit the new format.

File: collections/cams_glc_2017.yaml (hunk 3 above)
Comment (chorng) [reply collections/cams_glc_2017.yaml]: Same as above. Fixed.

**Comment by chorng** [comment]
Feel free to merge it, thanks a lot :)
